CRUD/service/connection.hpp

```diff
@@ -77,7 +77,7 @@ class connection
                 request_, buffer_.data(), buffer_.data() + bytes_transferred);
             auto itr = std::find_if( request_.headers.begin(), 
                                      request_.headers.end(), 
-                                     [](const header &h) { return h.name == "Content-Length" ; }
+                                     [](const header &h) { return h.name == "content-length" ; }
             ) ;
             if ( itr != request_.headers.end()) {
                 request_.data = std::string(data, boost::lexical_cast<long>(itr->value)) ;
```

Comment:
I normally change  CRUD directly here 
https://github.com/venediktov/CRUD 
then issue following command in vanilla-rtb 
git subtree pull --prefix CRUD git@github.com:venediktov/CRUD.git master --squash

So if you do that , we should get it changed in the external source , then merged into vanilla-rtb 
Let me know if  you can do it , if not I will accept your merge and later change the original repository.